website/versioned_docs/version-acs_cc/project/2025/alexandra.diaconu03/index.md

```diff
@@ -70,7 +80,7 @@ The **buzzer** emits sound cues for actions and is connected to a GPIO pin
 
 ![Top view](top-view_resized.webp)
 
-![Screen front](screen-front_resized.webp)

```

Comment:
Did you delete this image?

Reply:
I agree, but this file is not delete from the repository. Did you add this (delete) to the commit. You can check this with `git status`.